docs/getting_started.md

```diff
@@ -89,6 +89,9 @@ necessary dependencies
     * `source <path/to/.venv>/bin/activate`
 
 4. Upgrade pip and setuptools with `pip install --upgrade pip setuptools`
+
+** On Python3.8, there is a bug on numpy with setuptools 65.6.0 https://github.com/numpy/numpy/issues/22623
+WA : `pip install setuptools==65.5.0`
```

Comment:
Maybe worth to add snippet of the error which user will get when hitting the bug

Reply:
done